doc/source/examples/multiphysics/sloshing-in-rectangular-tank/sloshing-in-rectangular-tank.rst

```diff
@@ -0,0 +1,203 @@
+    set grid type          = subdivided_hyper_rectangle
+    set grid arguments     = 5, 2 : -1, -1 : 1, 0.1 : true
+    set initial refinement = 6
+  end
+
+
+The ``physical properties`` are mainly used to establish the Reynolds number of the sloshing liquid. For the air, however, the work of Carrica *et al.* `[1]  <https://onlinelibrary.wiley.com/doi/abs/10.1002/fld.1279>`_ does not give any physical properties. We thus fix the air to be significantly less dense than the liquid, but we keep its viscosity at a certain reasonable viscosity to ensure numerical stability.
+
+.. code-block:: text
+
+  #---------------------------------------------------
+  # Physical Properties
+  #---------------------------------------------------
+  
+  subsection physical properties
+    set number of fluids = 2
+    subsection fluid 0
+      set density             = 0.001
+      set kinematic viscosity = 0.001
+    end
+    subsection fluid 1
+      set density             = 1
+      set kinematic viscosity = 0.5
+    end
+  end
+
+
+The ``source term`` subsection is used to enable the gravitational acceleration along the :math:`y` direction.
+
+.. code-block:: text
+
+  # --------------------------------------------------
+  # Source term
+  #---------------------------------------------------
+  
+  subsection source term
+    set enable = true
+    subsection xyz
+      set Function expression = 0 ; -1 ; 0
+    end
+  end
+    
+
+
+-----------------------
+Running the simulation
+-----------------------
+
+We can call the gls_navier_stokes_2d for each Reynolds number. For :math:`Re=20`, this can be done by invoking the following command:
+
+.. code-block:: text
+
+  mpirun -np 8 gls_navier_stokes_2d sloshing-in-rectangular-tank_Re0020.prm
+
+to run the simulation using eight CPU cores. Feel free to use more.
+
+
+.. warning:: 
+    Make sure to compile lethe in `Release` mode and 
+    run in parallel using mpirun. This simulation takes
+    :math:`\approx` 8 minutes (Re=2) to 6 hours (Re=2000) on 8 processes.
+
+
+-------
+Results
+-------
+
+We compare the relative height of the free surface at :math:`x=0` with an analytical solution proposed by `Wu et al. [2] <https://link.springer.com/article/10.1023/A:1017558826258>`_. For the Reynolds number of 2, 20 and 200, data were directly extracted from `[1] <https://onlinelibrary.wiley.com/doi/abs/10.1002/fld.1279>`_, whereas for the Reynolds of 2000, the simplified analytical expression of Wu et al. is used. The results for Reynolds number of 2, 20, 200 and 2000 can be post-processed by invoking the following command from the folder of the Reynolds number of interest::
+
+.. code-block:: text
+
+  python3 ../sloshing_post_processing.py . sloshing-in-rectangular-tank_Re0020.prm
+
+in the folder where you have ran the simulation for a given Reynolds number (20 in the above example). You need to ensure that the ``lethe_pyvista_tools`` module included within Lethe is in your Python path.
```

Comment:
For the first sentence (half-sentence), you were kind of mentioning it already before the command
```suggestion
.. important::
    You need to ensure that the ``lethe_pyvista_tools`` module included within Lethe is in your Python path.
```